### PR TITLE
Add missing 1.6.1 and new 1.6.3 and 2.0.2

### PR DESCRIPTION
**Pull request by lagerspetz**
Add missing Spark versions: 1.6.1 and new Spark versions: 1.6.3 and 2.0.2 to spark_ec2.py

### CODE REVIEW

**Comment by shivaram** [comment]
Thanks @lagerspetz - LGTM. Merging this

File: spark_ec2.py [hunk 1]
```diff
@@ -76,10 +76,13 @@
     "1.5.1",
     "1.5.2",
     "1.6.0",
+    "1.6.1",
     "1.6.2",
+    "1.6.3,
```
Comment (wearpants) [inline spark_ec2.py:81]: You're missing a closing quote here. Found the hard way.

**Comment by wearpants** [comment]
There's a missing closing quote after 1.6.3.

Found the hard way.

File: spark_ec2.py (hunk 1 above)
Comment (shivaram) [reply spark_ec2.py:81]: Ah - Thanks @wearpants - My bad. I'll push a fix right now

Comment (shivaram) [reply spark_ec2.py:81]: Pushed a fix in https://github.com/amplab/spark-ec2/commit/5188c78a0e9e5d34164ccbe1a361aeaac653bcc0

**Pull request merged**